lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -6,12 +6,24 @@ module SharedValues
 
     class AndroidSdkUpdateAction < Action
       def self.run(params)
-        # Install Android-SDK via brew
-        require 'fastlane/plugin/brew'
+        # Install Android-SDK
         if FastlaneCore::Helper.mac?
+          require 'fastlane/plugin/brew'
           Actions::BrewAction.run(command: "cask ls --versions android-sdk || brew cask install android-sdk")
+          sdk_path = File.realpath("../..", FastlaneCore::CommandExecutor.which("android"))
```

Comment:
Now I would determine the sdk_path just with the `sdk_manager`

Reply:
Similar to above -- if someone has an old version of the sdk installed via homebrew, I think it's still useful to be able to detect and update it.

Reply:
Like stated before, we should be fine.
In addition, the `android` tool is marked deprecated, so it could be removed in new versions.

Reply:
done